cdi/4.1/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Jakarta Context Dependency Injection 4.1"
+date: 2022-05-30
+summary: "Release for Jakarta EE 11"
+---
+
+Jakarta Contexts Dependency Injection specifies a means for obtaining objects in such a way as to maximize reusability, testability and maintainability compared to traditional approaches such as constructors, factories, and service locators (e.g., JNDI).
+
+### New features, enhancements or additions
+<!-- List here -->
+Several minor issues in the Specification, APIs and TCK will be addresses as detailed in the [CDI 4.1 Milestone](https://github.com/jakartaee/cdi/milestone/1)
+
+### Removals, deprecations or backwards incompatible changes
+<!-- List here -->
+* The Expression Language SPI methods on the BeanManager will be deprecated in preparation for a move to a new subinterface
+that will be part of a new artifact. This removes the dependency from the Jakarta Expression Language core API, bot the
+clean-up dependencies and refactor features that are not client facing.
```

Comment:
```suggestion
clean-up dependencies and refactor features are not client facing.
```

Reply:
That whole sentence was awkward, so I have reworded it.